server/src/com/cloud/network/router/VirtualNetworkApplianceManagerImpl.java

```diff
@@ -1589,8 +1589,8 @@ protected StringBuilder createGuestBootLoadArgs(final NicProfile guestNic, final
     protected StringBuilder createRedundantRouterArgs(final NicProfile nic, final DomainRouterVO router) {
         final StringBuilder buf = new StringBuilder();
 
-        final long networkId = nic.getNetworkId();
-        final NetworkVO network = _networkDao.findById(networkId);
+        // final long networkId = nic.getNetworkId();
```

Comment:
While making changes in this file, I noticed that Eclipse was pointing out that the variable network was not in use. Instead of removing the entire line, I just commented it out in case someone later wanted to easily resume making use of this variable.

Once I commented it out, Eclipse pointed out that the variable networkId was no longer in use. Same story for it with regards to commenting it out.